### PR TITLE
review-feature/title-menu

### PR DESCRIPTION
**Pull request by elawrobel**
Menu and title - new components

I added two new components: title and menu. 
Menu should linked to subpages and tile is just random title for the while.

### CODE REVIEW

File: app/components/Header/NavBar.js [hunk 1]
```diff
@@ -2,4 +2,5 @@ import styled from 'styled-components';
 
 export default styled.div`
   text-align: center;
+  background-color: primaryDark;
```
Comment (ucanthurtme) [inline app/components/Header/NavBar.js]: it won't work that way 

Screenshot 2020-03-31 at 19 15 07

I've not clue how it resolve yet. Need to dig into how themes works inside react-boilerplate.

Comment (ucanthurtme) [reply app/components/Header/NavBar.js]: Here is the link to [docs](https://styled-components.com/docs/advanced)

File: app/components/Menu/messages.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/*
+ * HomePage Messages
+ *
+ * This contains all the text for the HomePage component.
+ */
+import { defineMessages } from 'react-intl';
+
+export const scope = 'boilerplate.components.Menu';
```
Comment (ucanthurtme) [inline app/components/Menu/messages.js]: it's not big deal however shouldn't we change it to charityActivity.components.Menu ?

File: app/components/Title/messages.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/*
+ * HomePage Messages
+ *
+ * This contains all the text for the HomePage component.
+ */
+import { defineMessages } from 'react-intl';
+
+export const scope = 'boilerplate.components.Header';
```
Comment (ucanthurtme) [inline app/components/Title/messages.js]: it's not big deal however shouldn't we change it to charityActivity.components.Menu ?

File: app/containers/NewApp/index.js [hunk 4]
```diff
@@ -7,13 +7,15 @@
  */
 
 import React from 'react';
-import { Helmet } from 'react-helmet';
+import { Helmet } from 'react-helmet'; // QA: dlaczego niektóre importy są w nawiasach a inne nie?
```
Comment (ucanthurtme) [inline app/containers/NewApp/index.js:10]: I've refered to the question in one of the previous answers. It's named import. Additionally to the previously send url, you can read all MDN sections regarding exports and imports.

File: app/components/Menu/index.js [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    color: #0070ad;
+  }
+`;
+
+function Menu() {
+  return (
+    <div>
+      <Nav>
+        <NavItem>
+          <NavLink to="/">
```
[the remark this answers is not in the document]
Comment (elawrobel) [reply app/components/Menu/index.js:42]: so i can't it use in this way?

[the remark this answers is not in the document]
Comment (ucanthurtme) [reply app/components/Menu/index.js:42]: You can, it's just reminder

File: app/components/Header/NavBar.js (hunk 1 above)
Comment (ucanthurtme) [reply app/components/Header/NavBar.js]: Is it fixed ? Do you need any help ?

Comment (elawrobel) [reply app/components/Header/NavBar.js]: It's fixed :)

**Pull request merged**